src/types.ts

```diff
@@ -271,6 +272,7 @@ export const serverPrefKeys: (keyof ServerPreferences)[] = [
   'hoursSchedule',
   'SMTPConfig',
   'systemManagerPermissionName',
+  'managerCanEditLookupTables',
```

Comment:
Add a preference for the off chance that we *don't* want the system manager to access lookup tables (it's `true` by default)